### ISSUE TITLE
I think the groupId, artifactId are wrong in the README

### ISSUE THREAD

**Issue by amalagaura** [badoualy/kotlogram#4]
Jitpack says it is:

	        compile 'com.github.badoualy.kotlogram:api:0.0.5'

Which does work. The combination in the README gives an empty jar.

**Comment by badoualy** [comment]
Yes, empty jar is "normal" => it just contains a pom.xml using the different modules of Kotlogram :)

**Comment by amalagaura** [comment]
Thanks for your work on this. I see what you mean with the master project. I did have some experience with maven and poms a few years ago and creating these empty master projects.

I think the issue for me in Intellij was after it loaded the jar it could not decompile and could not resolve any imports so I thought it was broken. Apparently different version of the Kotlin compiler. I recreated the project and modified the build.gradle with 1.0 of Kotlin and deployed locally so I could resolve imports and compile. 

Anyway thank you for this.
